src/test/components.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import Home from '../components/Home';
+import AllCrypto from '../components/AllCrypto';
+import CryptoDetails from '../components/CryptoDetails';
+
+describe('Test Components', () => {
+  test('test Home', () => {
+    const tree = renderer
+      .create(<Provider store={store}><Home /></Provider>)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test('test AllCrypto', () => {
+    const tree = renderer
+      .create(<Provider store={store}><AllCrypto /></Provider>)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test('test CryptoDetails', () => {
```

Comment:
- [ ] Good job writing tests for your components 💯 Kindly fix your failing tests
- Your site:
![Annotation 2022-06-24 165114](https://user-images.githubusercontent.com/85113567/175574309-c91ae390-014c-4b77-93a6-3ae7c392c0dd.jpg)

Reply:
* Addresssed 👌